errors/src/commonMain/kotlin/dev/icerock/moko/errors/presenters/ErrorPresenter.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.errors.presenters
+
+import dev.icerock.moko.errors.ErrorEventListener
+import dev.icerock.moko.mvvm.dispatcher.EventsDispatcher
+
+expect abstract class ErrorPresenter<T : Any>() : ErrorPresenterBase<T>
+
+@Suppress("UnnecessaryAbstractClass")
```

Comment:
now suppress not required (you already have abstract field)

Reply:
Fixed.